draft-ietf-quic-transport.md

```diff
@@ -1483,10 +1483,13 @@ subsequent sections.
 ## STREAM Frame {#frame-stream}
 
 STREAM frames implicitly create a stream and carry stream data. The type byte
-for a STREAM frame contains embedded flags, and is formatted as `11FSSOOD`.
+for a STREAM frame contains embedded flags, and is formatted as `1UFSSOOD`.
 These bits are parsed as follows:
 
-* The first two bits must be set to 11, indicating that this is a STREAM frame.
+* The first bit must be set to 1, indicating that this is a STREAM frame.
+
+* 'U' is the UNI bit, which is used to indicate this is a unidirectional
+  stream, and no stream frames may be sent in response.
```

Comment:
Why wouldn't it? Consider the strawman WebRTC structure I proposed where you just transmit packets on whatever stream is currently idle.

Another example would be any protocol where the server speaks first, like, say IMAP. If you just think of each QUIC stream as like a TCP connection, then the server might send the server greeting upon initial connection, in which case it wouldn't need to wait for any data on implicit streams.

I'm not saying you can't prohibit this in a protocol mapping, but it's a new restriction you've created that has to be dealt with